hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreCompactor.java

```diff
@@ -104,9 +105,9 @@ public boolean start() throws IOException {
         COMPACTING_MEMSTORE_TYPE_DEFAULT);
 
     switch (t) {
-      case 1: type = Type.COMPACT_TO_SKIPLIST_MAP;
+      case 1: type = ActionUponInvocation.COMPACT_TO_SKIPLIST_MAP;
```

Comment:
Where does the value of t come from? Is this some programmer-level env flag? Let's do some cleanup here ...